lr_face/data_providers.py

```diff
@@ -117,6 +117,69 @@ def enfsi_data(resolution, year) -> PairsWithIds:
     return PairsWithIds(pairs=X, is_same_source=y, pair_ids=ids)
 
 
+def lfw_data(resolution) -> PairsWithIds:
+    folder = os.path.join('resources', 'lfw')
+    df = pd.read_csv(os.path.join(folder, 'pairs.txt'), sep="\t", names=['col1', 'col2', 'col3', 'col4'])
+    
+    num_images_batch = int(df.loc[0]['col2'])
```

Comment:
dit is niet een goede manier van parameters doorspelen. Er wordt echter ook niks met deze parameters gedaan - alleen het product wordt gebruikt.
I suggest dropping pairs.txt and rewriting this code to read all images with the corresponding person ids (see PR on forenface), and returning this as an ImagesWithIds.

Reply:
if that proves too slow a generator may work

Reply:
or limiting the number of images read